src/plugins/kibana_react/public/saved_objects/saved_object_finder.tsx

```diff
@@ -210,7 +207,7 @@ class SavedObjectFinderUi extends React.Component<
   public render() {
     return (
       <React.Fragment>
-        {this.renderSearchBar()}
+        {false && this.renderSearchBar()}
```

Comment:
I have to agree with @majagrubic , we definitely need the type filter here. It shouldn't be too hard to add this, looks like we would have to render just the filter dropdown in https://github.com/elastic/kibana/pull/55342/files#diff-be19b10719b741707a5d1e1d5fef6a65R494